senza/components/auto_scaling_group.py

```diff
@@ -183,6 +190,111 @@ def component_auto_scaling_group(definition, configuration, args, info, force, a
     return definition
 
 
+def normalize_network_threshold(threshold):
+    unit = "Bytes"
+    if threshold is None:
+        return []
+    if isinstance(threshold, int):
+        return [str(threshold), unit]
+    amount = 1024
+    shortcuts = {
+        "B": "Bytes",
+        "KB": "Kilobytes",
+        "MB": "Megabytes",
+        "GB": "Gigabytes",
+        "TB": "Terabytes"
+    }
+    try:
+        # if someone write just Threshold: 10
+        amount = int(threshold)
+        return [threshold, unit]
+    except ValueError:
+        # check if there is a space as if somebody wrote Threshold: 20 GB
+        if " " in threshold:
+            # okay, so split it
+            [amount, unit] = threshold.split(" ")
```

Comment:
just remove the brackets on the left side and use threshold.split() (without param which will split on 1+ whitespace chars)